enrollment-server-onboarding-common/src/main/java/com/wultra/app/onboardingserver/common/configuration/OnboardingComponentsConfiguration.java

```diff
@@ -39,14 +38,6 @@
  */
 @Configuration
 @ConditionalOnProperty(name = "enrollment-server.onboarding.enabled", havingValue = "true")
-@EnableJpaRepositories(basePackages = {
```

Comment:
And how are entities and daos from common initialized now? I mean for example `com.wultra.app.onboardingserver.common.database.OnboardingOtpRepository`?

Reply:
Now I think that `OnboardingComponentsConfiguration` should be moved to downstream and property `enrollment-server.onboarding.enabled` should be removed.

Reply:
We can discuss on grooming, we never needed those annotations.

Reply:
Just for explanation: It is needed in the downstream project after the split.